content/tutorials/manage-the-rippled-server/configuration/propose-amendments.md

```diff
@@ -0,0 +1,111 @@
+---
+html: propose-amendments.html
+parent: configure-rippled.html
+blurb: 
+labels:
+  - Blockchain
+---
+# Contribute Code to the XRP Ledger
+
+The software that powers the XRP Ledger is open-source, so anyone can download, modify, extend, or explore it.
+
+
+## Standards Draft
+
+If you intend to make significant changes to the ledger, write a standards draft detailing the changes and submit it to the **XRPL-Standards** repository. Give the community time to discuss your idea and incorporate feedback before committing time and resources to developing the code.
+
+See: [Standard Proposals](https://github.com/XRPLF/XRPL-Standards/discussions/categories/standard-proposals).
+
+**Note:** Bug fixes don't require a standards draft.
+
+
+### Naming and Numbering Conventions
+
+Standard drafts must be named and numbered in this format:
+
+`4-digit natural number with leading 0s` XLS-`natural number without leading zeroes.`d: `Title of Proposal`
+
+**Example:** 0032 XLS-32d: Request URI Structure
+
+
+### Formatting Conventions
+
+Your standards draft should supply the community with the necessary information to facilitate useful discussion. Helpful sections to include are:
+
+- Abstract/Introduction
+- Specifications
+- Considerations
+
+
+<!-- This hasn't been updated since 2021; is this step from the README valid? -->
+When a standard moves to a folder + file(s) in the Code section of this repository, it will be added to the standards.toml file: https://github.com/XRPLF/XRPL-Standards/blob/master/standards.toml
+
+<!-- How much is enough discussion? The README mentions creating an issue to reference in the PR, but this doesn't seem to be happening in practice. -->
+After sufficient input from the community, create a corresponding **Issue** with the finalized standards draft in `XRPLF:develop`.
```

Comment:
There is a cleanup job to do here, and a PR open to change the README. I believe the current praxis does not clearly define when a draft is not a draft anymore etc., so a lifecycle for the XLSd's are necessary.

https://github.com/XRPLF/XRPL-Standards/pull/87